addons/web/static/src/views/view_dialogs/export_data_dialog.js

```diff
@@ -408,9 +408,18 @@ export class ExportDataDialog extends Component {
     }
 
     async setDefaultExportList() {
-        this.state.exportList = Object.values(this.knownFields).filter(
-            (e) => e.default_export || this.props.defaultExportList.find((i) => i.name === e.id)
+        const defaultExportList = this.props.defaultExportList
+            .map((defaultField) => {
+                const field = this.knownFields[defaultField.name];
+                return field && field.id === defaultField.name ? field : null;
```

Comment:
Is the `field.id === defaultField.name` necessary?? I don't undertstand why it could be.

If not:
```suggestion
                return this.knownFields[defaultField.name];
```

Reply:
@aab-odoo changes have been done.
Thank you.